src/init_interp/initInterp.F90

```diff
@@ -455,6 +457,25 @@ subroutine initInterp (filei, fileo, bounds, glc_behavior)
           CYCLE
        end if
 
+       !---------------------------------------------------
+       ! Get scale_by_thickness value (.true./.false.) from output
+       ! file to avoid backwards compatibility issue.
+       ! TODO status before the next pio_get_att returns 0; however, after
+       !      the call, the model just crashes when the attribute is missing.
+       !      How to check for the attribute to avoid
+       !      hardwiring the variable names below? Ideas:
+       !      - call read_issue_fixed_metadata?
+       !---------------------------------------------------
+
+       scale_by_thickness = .false.  ! default value
+
+       if  ( trim(varname) == 'H2OSOI_LIQ' .or. trim(varname) == 'H2OSOI_ICE') then
```

Comment:
I added `SMP` and `HK` to the if-statement for the next commit; 
however, @billsacks I would rather not hardwire the variable names, as explained in the TODO comment starting in line 463 above. I just realized that we could also resolve this by making scale_by_thickness a required attribute, as you suggested [here](https://github.com/ESCOMP/CTSM/pull/1301/files#r594826858).

Reply:
Ok. I'm adding scale_by_thickness=.true. or .false to all variables with dim2name=
**levmaxurbgrnd levgrnd levtot levsno levsno1 levlak**
but not
**levcan vegwcs pft lat numrad**
Pls let me know if - off the top of your head - any item in these lists seems wrong.

Reply:
Off the top of my head, that looks like a reasonable list. I think levlak uses the default multilevelCopy, so probably doesn't need it for now. (I don't think we currently handle changing lake levels in init_interp, though in principle we could.)

Reply:
You will find that I modified that orig. list for the code to work correctly.